consensus/hotstuff/votecollector/interfaces.go

```diff
@@ -0,0 +1,74 @@
+package votecollector
```

Comment:
A number of auditors >= 1 will read this code and will want to make 💯  sure they understand where the verification of each and every signature is done.

How do we tell them where that verification responsibility lies more precisely in this file? I think a good comment written today can save us a long meeting tomorrow.

(this applies at least to `TrustedAdd` and `QCBuilder`)